pom.xml

```
@@ -288,7 +287,7 @@
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
         <artifactId>bom-${jenkins.baseline}.x</artifactId>
-        <version>4051.v78dce3ce8b_d6</version>
+        <version>4228.v0a_71308d905b_</version>
```

This would downgrade jsoup to 1.18.3. So let's use the first BOM with 1.19.1

```suggestion
        <version>4488.v7fe26526366e</version>
```